### ISSUE TITLE
Agregar ejercicios programables sobre REST

### ISSUE THREAD

**Issue by flbulgarelli** [MumukiProject/mumuki-contenido#16]
@NadiaFinzi  dijo:

> Para la lección 2 de Redes falta hacer dos ejercicios de REST.

**Comment by NadiaFinzi** [comment]
@aguspina hizo [este ejercicio](https://bibliotheca.mumuki.io/#/guides/mumukiproject/mumuki-guia-text-aplicaciones-en-internet/exercises/11). No sé si apuntás a hacer otros dos con api propia o si está bien con eso.

**Comment by flbulgarelli** [comment]
No, apunto a hacer ejercicios en los que haya que programar con Ruby y `RestClient`
